databricks/koalas/indexes.py

```diff
@@ -108,13 +108,26 @@ class Index(IndexOpsMixin):
     Index(['a', 'b', 'c'], dtype='object')
     """
 
-    def __init__(self, data: Union[DataFrame, list], dtype=None, name=None) -> None:
+    def __new__(cls, data: Union[DataFrame, list], dtype=None, name=None, names=None):
+        if isinstance(data, list) and all([isinstance(item, tuple) for item in data]):
+            # MultiIndex
+            return MultiIndex.from_tuples(data, names=names)
```

Comment:
Is this a right usage of `__new__`?
I think it's supposed to just create an instance, and the initialization should be in `__init__`.

Reply:
Hmm... actually I just referred to this kind of behavior from pandas.

```python
    def __new__(
        cls, data=None, dtype=None, copy=False, name=None, tupleize_cols=True, **kwargs
    ) -> "Index":

        ...

        # range
        if isinstance(data, RangeIndex):
            return RangeIndex(start=data, copy=copy, dtype=dtype, name=name)
        elif isinstance(data, range):
            return RangeIndex.from_range(data, dtype=dtype, name=name)
```


Let me investigate more about the usage of `__new__`.

Reply:
ah, I see. pandas rather doesn't use `__init__` at all.

Reply:
It's right. what do you think about we also follow that behavior ??

Reply:
Let's follow the behavior.